README.md

```diff
@@ -37,7 +37,7 @@ From the `server` directory:
 - `./gradlew test` to test the server
 - `./gradlew checkstyleMain` to run Checkstyle on the server Java code in the `src/main` folder
 - `./gradlew checkstyleTest` to run Checkstyle on the server Java code in the `src/test` folder
-- `./gradlew check` will run both the tests and Checkstyle in one command
+- `./gradlew check` will run the tests, run the Checkstyle, and generate coverage reports in one command
```

Comment:
```suggestion
- `./gradlew check` will run the tests, run the Checkstyle checks, and generate coverage reports in one command
```

Reply:
Note that these changes have nothing to do with `@angular/flex-layout` and probably should have been in another PR. 😜